product_packaging_dimension/models/product_packaging.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    max_weight = fields.Float("Weight (kg)")
+    # lngth IS NOT A TYPO https://github.com/odoo/odoo/issues/41353
+    lngth = fields.Integer("Length (mm)", help="length in millimeters")
+    width = fields.Integer("Width (mm)", help="width in millimeters")
+    height = fields.Integer("Height (mm)", help="height in millimeters")
```

Comment:
I am OK for https://github.com/OCA/product-attribute/pull/531#discussion_r355553682 but can you add docsting to prevent unit change and explain the choice :wink:

Reply:
Done in 14a0a32